src/foam/nanos/column/ColumnConfigPropertyValue.js

```diff
@@ -81,7 +81,8 @@ foam.CLASS({
               if ( p instanceof foam.core.AbstractFObjectPropertyInfo ) {
                 cls = p.getValueClass();
               } else {
```

Comment:
have no idea why it looked like that. maybe I reverted something on my columnconfig branch or something